### DISCUSSION TITLE
WARNING:root:NaN or Inf found in input tensor.

### DISCUSSION THREAD

**Discussion by stalevna** [coqui-ai/TTS#1500] [General Q&A]
Hi,
Has anyone had such warnings when training VITS TTS?
Because of it, I can't see many output parameters, they are all nan :(
for example
    avg_loss_gen: nan (+nan)

I am at a loss what to do, please, help

**Comment by Ca-ressemble-a-du-fake** [comment]
Hi @stalevna did you find a solution to this ? I appeared all of a sudden during training, and I am surprised the training could continue. Should I stop the current run ?
